UET/Redpoint.Uet.Configuration/BuildConfigTargetPlatformConverter.cs

```diff
@@ -0,0 +1,68 @@
+﻿namespace Redpoint.Uet.Configuration
+{
+    using System;
+    using System.Collections.Generic;
+    using System.Text.Json.Serialization;
+    using System.Text.Json;
+
+    public class BuildConfigTargetPlatformConverter : JsonConverter<BuildConfigTargetPlatform[]>
+    {
+        public override BuildConfigTargetPlatform[] Read(ref Utf8JsonReader reader, Type typeToConvert, JsonSerializerOptions options)
+        {
+            if (reader.TokenType != JsonTokenType.StartArray)
+                throw new JsonException("Expected StartArray token");
+
+            var platforms = new List<BuildConfigTargetPlatform>();
+
+            while (reader.Read() && reader.TokenType != JsonTokenType.EndArray)
+            {
+                if (reader.TokenType == JsonTokenType.String)
+                {
+                    platforms.Add(new BuildConfigTargetPlatform()
```

Comment:
Styling - Omit parenthesis for parameterless constructor, including down below as well.

Reply:
Only removed parenthesis in that instance, down below doesn't have initializers, so parentheses are required.